### PR TITLE
Convert python scripts to be executed using Python3

### PR DESCRIPTION
**Pull request by ovlords**

### CODE REVIEW

File: provisioners/puppet/modules/config/data/RedHat/Amazon-2.yaml [hunk 1]
```diff
@@ -17,16 +17,16 @@ config::package_manager_packages:
   - name: inspec
     version: 1.51.6
     provider: puppet_gem
+  - name: python.x86_64
+  - name: python2-pip.noarch
+  - name: python2-cheetah.x86_64
```
Comment (mbloch1986) [inline provisioners/puppet/modules/config/data/RedHat/Amazon-2.yaml:22]: Do we need those python2 packages ?

Comment (mbloch1986) [reply provisioners/puppet/modules/config/data/RedHat/Amazon-2.yaml:22]: Same for the other OS

File: provisioners/puppet/modules/config/manifests/base.pp [hunk 2]
```diff
@@ -142,30 +132,35 @@
...
   }
 
-  package { [ $python_package, $python_pip_package, $python_cheetah_package, $python_alt_package, ]:
-    ensure => latest,
-  }
-
   package { [ 'requests', 'retrying', 'sh' ]:
     ensure   => latest,
-    provider => 'pip',
+    # provider => 'pip',
```
Comment (mbloch1986) [inline provisioners/puppet/modules/config/manifests/base.pp:137]: Please remove the comments !

**Review by mbloch1986 — CHANGES_REQUESTED** [review]
See the comments and please update the Makefile to use python3 as well. Thanks !

File: provisioners/puppet/modules/config/manifests/base.pp (hunk 2 above)
Comment (ovlords) [reply provisioners/puppet/modules/config/manifests/base.pp:137]: Removed

File: provisioners/puppet/modules/config/data/RedHat/Amazon-2.yaml (hunk 1 above)
Comment (ovlords) [reply provisioners/puppet/modules/config/data/RedHat/Amazon-2.yaml:22]: Did we want to keep them for backwards compatibility?

Comment (ovlords) [reply provisioners/puppet/modules/config/data/RedHat/Amazon-2.yaml]: Removed

**Pull request merged**